packages/vue/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "strict": true,
     "jsx": "preserve",
     "importHelpers": true,
-    "declaration": true,
+    "declaration": false,
```

Comment:
This fixes the issue with `script setup` and having an `interface` in it, and getting an error message.